assets/_scss/elements/_buttons.scss

```diff
@@ -0,0 +1,77 @@
+// Buttons
+
+input[type="button"],
+input[type="submit"],
+input[type="reset"],
+input[type="image"],
+button[type="button"],
+button[type="submit"],
+button[type="reset"] {
+  -webkit-appearance: none;
```

Comment:
Remove all prefixes and replace with what would be in auto prefixer:

https://github.com/postcss/autoprefixer/wiki/support-list